WebRoot/js/zimbraMail/share/model/ZmSettings.js

```diff
@@ -1203,11 +1203,12 @@ ZmSettings.prototype._showConfirmDialog = function(text, callback, style) {
 
 ZmSettings.prototype._implicitChangeListener =
 function(ev) {
-	if (!appCtxt.get(ZmSetting.OPTIONS_ENABLED)) {
+	var id = ev.source.id;
+
+	if (!appCtxt.get(ZmSetting.OPTIONS_ENABLED) && ZmSetting.FOLDERS_EXPANDED !== id) {
```

Comment:
better to move single if condition to line no 1219
after we have done processing for expand/collapse folder save

Reply:
ok